src/bindings/python/docs/test_examples.md

```diff
@@ -24,6 +28,12 @@ To run OpenVINO Python API 1.0 tests, use this command:
 pytest tests_compatibility/
 ```
 
+By default, tests are being run on the CPU plugin. If you want to run them on another plugin, 
```

Comment:
By default, tests are run on the CPU plugin. If you want to run them on a different plugin,

Reply:
done